core/src/main/java/com/nextdoor/bender/handler/BaseHandler.java

```diff
@@ -373,6 +374,8 @@ public void run() {
       try {
         String raw = null;
         raw = this.ser.serialize(this.wrapper.getWrapped(ievent));
+        Stat bytesSent = new Stat("serializer.serialized_bytes", raw.getBytes(StandardCharsets.UTF_8).length);
```

Comment:
Can we get pre-transformation and post-transformation bytes?

Reply:
Sure I'll add that in.